package.json

```diff
@@ -100,19 +99,13 @@
     "eslint-plugin-jest": "^27.2.3",
     "generate-password": "^1.7.0",
     "glob": "^10.1.0",
-    "gulp": "^4.0.2",
-    "gulp-postcss": "^9.0.1",
-    "gulp-replace": "^1.1.4",
-    "gulp-sass": "^5.1.0",
-    "gulp-sourcemaps": "^3.0.0",
     "husky": "^8.0.3",
     "jest": "^29.5.0",
     "jest-transform-stub": "^2.0.0",
     "lint-staged": "^13.2.3",
-    "nasawds": "^3.0.338-beta.1",
+    "nasawds": "github:lpsinger/nasawds#uswds-3.1.0-npm-scripts-prepare",
```

Comment:
what are you working around here with this fork instead of a version?

Reply:
looks like a future version that's not available yet

Reply:
Correct. That branch combines the following two PRs: https://github.com/bruffridge/nasawds/pull/13, https://github.com/bruffridge/nasawds/pull/14

Reply:
All upstream PRs are merged now.